### DISCUSSION TITLE
How can I override the mappings for snippet placeholders?

### DISCUSSION THREAD

**Discussion by zeitchef** [VonHeikemen/lsp-zero.nvim#33] [Q&A] [answered]
I've got `<C-h>` and `<C-l>` deeply ingrained in my soul to navigate back/forth inside snippets. What's the easiest way to override the default keymaps? Thanks!

**Comment by VonHeikemen** [comment, marked as answer]
You would "expose" the default mappings, then create the new ones based on the existing mapping. Finally, pass your new set of mappings to `.setup_nvim_cmp()`

```lua
local lsp = require('lsp-zero')
lsp.preset('recommended')

local cmp_mapping = lsp.defaults.cmp_mappings()
cmp_mapping['<C-h>'] = cmp_mapping['<C-b>']
cmp_mapping['<C-l>'] = cmp_mapping['<C-d>']

lsp.setup_nvim_cmp({
  mapping = cmp_mapping
})

lsp.setup()
```

**Reply by zeitchef** [reply]
Thank you!
